_posts/2024-01-17-January.md

```diff
@@ -0,0 +1,249 @@
+---
+layout: post
+title: "Top of the Year in Minetest (16)"
+contributors:
+  - Zughy
+  - rubenwardy
+  - jordan4ibanez
+  - Bituvo
+  - Apercy
+  - BuckarooBanzay
+  - jdiego
+authors:
+  - MisterE
+description: >
+  Desc Here
+tags:
+  - last_month
+image: /static/blog/2024_TopOfYear/header.jpg
+
+# forum_topic: https://forum.minetest.net/viewtopic.php?f=18&t=29889
+
+# pulls: https://github.com/minetest/minetest/pull
+gallery_apercy:
+- src: /static/blog/2024_TopOfYear/vwbug.png
+  caption: Lovebug 🪲
+- src: /static/blog/2024_TopOfYear/savoia.jpg
+  caption: Savoia S21 from "Porco Rosso"
+- src: /static/blog/2024_TopOfYear/ww1plane.jpg
+  caption: WW1 planes
+gallery_BlEx:
+- src: /static/blog/2024_TopOfYear/better_details.png
+  caption: Better Details page
+- src: /static/blog/2024_TopOfYear/featuredschems.png
+  caption: Featured Schematics
+- src: /static/blog/2024_TopOfYear/login_options.png
+  caption: More Login Options
+gallery_temple:
+- src: /static/blog/2024_TopOfYear/Mortuary_Temple_of_Ramesses_III_at_Medinet_Habu_1.png
+  caption: Overhead view
+- src: /static/blog/2024_TopOfYear/Mortuary_Temple_of_Ramesses_III_at_Medinet_Habu_2.png
+  caption: Inside
+- src: /static/blog/2024_TopOfYear/Mortuary_Temple_of_Ramesses_III_at_Medinet_Habu_3.png
+  caption: The Wall
+---
+
+Desc Here
+
+<!-- more -->
+
+- [Engine News](#engine-news)
+- [Games News](#games-news)
+- [Mods News](#mods-news)
+- [Art and Builds](#art-and-builds)
+- [In Other News](#in-other-news)
+
+## Engine News
+
+### New core dev: Lars
```

Comment:
Lars is ambigous, should call him luatic according to https://irc.minetest.net/minetest-dev/2024-01-12#i_6145553